product-mini/platforms/nuttx/main.c

```diff
@@ -296,6 +378,8 @@ main(int argc, char *argv[])
         goto fail3;
     }
 
+    if (is_repl_mode)
+        app_instance_repl(wasm_module_inst);
     if (func_name)
```

Comment:
Should modify to else if (func_name) like Linux?

Reply:
Fixed